styles.css

```diff
@@ -1097,6 +1097,20 @@ svg {
     line-height: 24px;
   }
 
+  .desktop_about {
+    /* padding: 10em 0; */
```

Comment:
- [ Required ] Good job implementing the changes from the previous reviewer, however, while doing so you introduced commented code in your repository. Please consider removing any pieces of commented code in your repository this is because commented code makes your repository look unprofessional.